src/ScriptEditor/ui/ScriptEditorRoot.tsx

```diff
@@ -313,8 +314,7 @@ function Root(props: IProps): React.ReactElement {
         },
       });
     }
-    //unmounting the editor will dispose all, doesnt hurt to dispose on close aswell
-    closingScript.model.dispose();
```

Comment:
If I'm reading this right, script files are now meant to exist as models permanently. Text files, however, are not: They get models when they are opened. These models aren't ever disposed of, though.